static/javascript/launch.js

```diff
@@ -174,7 +178,7 @@ function setLaunchType(launchType) {
   if (launchType === "cir" || launchType === "remote" || launchType === "url") {
```

Comment:
This line could be made a bit simpler by doing something like:
if (["cir", "remote", "url"].includes(launchType)) {

Reply:
Updated 👍